src/Schema.php

```diff
@@ -49,7 +49,7 @@ public static function make($json): Schema
         // @codeCoverageIgnoreStart
         throw new \InvalidArgumentException(sprintf(
             "No schema type available for %s",
-            $schema->type
+            $json->type
```

Comment:
FWIW, I'd love to have tests for this. It's a total 🏴 that this is coverage ignored.